src/provider/model/ContextChannel.ts

```diff
@@ -81,7 +80,7 @@ export class DesktopContextChannel extends ContextChannelBase {
         this._context = null;
     }
 
-    public serialize(): DesktopChannelTransport {
-        return {name: this.name, color: this.color, ...super.serialize() as {id: string, type: 'desktop'}};
+    public serialize(): SystemChannelTransport {
+        return {visualIdentity: this.visualIdentity, ...super.serialize() as {id: string, type: 'system'}};
```

Comment:
Worth doing `visualIdentity: {...this.visualIdentity}`?

Reply:
Yea, probably.